javascript/builtins/SharedArrayBuffer.json

```diff
@@ -0,0 +1,222 @@
+{
+  "javascript": {
+    "builtins": {
+      "SharedArrayBuffer": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer",
```

Comment:
the table has 2 subfeatures, which are not included here. Weirdly, the subfeatures have lower version-added than the basic support.

Reply:
`slice` is there and I'm adding SAB in DataView. The lower versions come through the preference situation, having the preference data will make it more clear.